protocol/contracts/v2/TreasuryReservesVault.sol

```diff
@@ -181,20 +181,11 @@ contract TreasuryReservesVault is ITreasuryReservesVault, TempleElevatedAccess {
         StrategyConfig storage strategyConfig = strategies[strategy];
         strategyConfig.underperformingEquityThreshold = underperformingEquityThreshold;
 
-        mapping(IERC20 => uint256) storage _tokenCredits = strategyTokenCredits[strategy];
-
-        uint256 _ceiling;
-        IERC20 _token;
+        ITempleStrategy.AssetBalance memory _assetBalance;
         uint256 _length = debtCeiling.length;
         for (uint256 i; i < _length; ++i) {
-            _token = IERC20(debtCeiling[i].asset);
-            _ceiling = debtCeiling[i].balance;
-
-            // Note: This will be redundant when https://github.com/yAudit/temple-lending-report/issues/11 is addressed.
-            // Revert if the debt ceiling + credit overflows
-            _checkAddOverflow(_ceiling, _tokenCredits[_token]);
-
-            strategyConfig.debtCeiling[_token] = _ceiling;
+            _assetBalance = debtCeiling[i];
```

Comment:
The overflow checks are no longer required here, because when strategies are removed the debt ceiling and credits is also deleted for that strategy|token pair